docs/moisture_dataset.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
```

Comment:
Can you add a note at the top of the file that explains what this script does?

Reply:
Done